Dockerfile

```
@@ -15,7 +15,20 @@ RUN yarn build
 CMD [ "yarn", "dev" ]
 
 FROM nginx:stable-alpine as prod
+# set working directory
+WORKDIR /syncmaster_ui
+
+# install app dependencies
+RUN npm cache clean --force \
+    npm install --global yarn
+
+COPY package*.json .
+RUN npm ci
+COPY . .
+# Replace URL in the prod image
+RUN sed -i "s#http://localhost:8000/#/#g" /syncmaster_ui/src/shared/api/utils.ts
```

Предлагаю сделать так:

```suggestion
ARG API_URL=/api
RUN sed -i "s#http://localhost:8000#$API_URL#g" /syncmaster_ui/src/shared/api/utils.ts
```

Так можно будет при необходимости собрать образ с другим адресом, просто передав `--build-arg API_URL=...` в момент сборки.
А слэш вернуть как префикс data provider

---

вернул

---

решил убрать ARG API_URL=/api а сделал ARG API_URL чтобы явно при сборке всегда указывать, так мне кажется лучше

---

Ответил в комменте ниже